.env.production

```diff
@@ -4,4 +4,5 @@ ENVIRONMENT=PROD
 API_URL_PREFIX='https://app-backend.k8s.test.cd.teamdigitale.it'
 MIXPANEL_TOKEN='b5c28d653684157f831d6b604d5e1956'
 ENABLE_TEST_IDP=NO
+GCM_SENDER_ID='317157111831'
```

Comment:
I think we will use a different one for production - @gunzip ?

Reply:
AFAIK Google does not require different sender for test / production and Apple requires different keys (not sender id) (but I'm not an expert)